gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #206: prompt the user to search on homepage

### Description

Prompt the user to search on homepage.

### Testing Directions

```
./setup
python manage.py runserver
```

1.  Navigate to `/`. Notice how it says "Howdy, Vik. Head over to Events to find shows you may be interested in."
1.  Navigate to `/events`. Mark an event as "Interested".
1.  Navigate to `/`. Notice how it says "Howdy, Vik. Let's get you dancing ASAP."

Repeat the above with only 1 event marked as "Going". Then repeat again with only 1 event marked as "Interested" and only 1 event marked as "Going".

### Checklist
